docs/user/ppl/functions/json.rst

```diff
@@ -49,14 +50,73 @@ Return type: BOOLEAN/DOUBLE/INTEGER/NULL/STRUCT/ARRAY
-    |---------------------|---------------------------------|-------------------------|
-    | json nested object  | {"a":"1","b":{"c":"2","d":"3"}} | {a:"1",b:{c:"2",d:"3"}} |
-    | json object         | {"a":"1","b":"2"}               | {a:"1",b:"2"}           |
-    | json array          | [1, 2, 3, 4]                    | [1,2,3,4]               |
-    | json scalar string  | "abc"                           | "abc"                   |
-    | json empty string   |                                 | null                    |
-    +---------------------+---------------------------------+-------------------------+
+    os> source=json_test | where json_valid(json_string) | eval json=json(json_string) | fields test_name, json_string, json
+    fetched rows / total rows = 6/6
+    +--------------------+--------------------------------------+-------------------------------------------+
+    | test_name          | json_string                          | json                                      |
+    |--------------------+--------------------------------------+-------------------------------------------|
+    | json nested object | {"a":"1","b":{"c":"2","d":"3"}}      | {'a': '1', 'b': {'c': '2', 'd': '3'}}     |
+    | json nested list   | {"a":"1","b":[{"c":"2"}, {"c":"3"}]} | {'a': '1', 'b': [{'c': '2'}, {'c': '3'}]} |
+    | json object        | {"a":"1","b":"2"}                    | {'a': '1', 'b': '2'}                      |
+    | json array         | [1, 2, 3, 4]                         | [1,2,3,4]                                 |
+    | json scalar string | "abc"                                | abc                                       |
+    | json empty string  |                                      | null                                      |
+    +--------------------+--------------------------------------+-------------------------------------------+
+
+JSON_EXTRACT
+____________
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_extract(doc, path[, path[, path])` Extracts a JSON value from a json document based on the path(s) specified.
+
+Argument type: STRING, STRING[, STRING[, STRING]]
+
+Return type: STRING/BOOLEAN/DOUBLE/INTEGER/NULL/STRUCT/ARRAY
+
+- Up to 3 paths can be provided, and results of all possible `path`s will be returned in an ARRAY.
+- If only one `path` is provided, returns an ARRAY if `path` points to multiple results (e.g. $.a[*]) or if the `path` points to an array.
+- Return null if `path` is not valid, or if JSON `doc` is MISSING or NULL.
+- If multiple paths are provided with paths that are not valid, will return an ARRAY where results of invalid paths are null.
+- Throws SemanticCheckException if `doc` or any `path` is malformed.
+- Throws ExpressionEvaluationException if any `path` is missing.
+
+Example::
+
+    os> source=json_test | where json_valid(json_string) | eval json_extract=json_extract(json_string, '$.b') | fields test_name, json_string, json_extract
```

Comment:
Does it accept "legacy" path (path which starts with ".")? if yes, this should be tested and documented. If no, this should be documented.